aspnet/fundamentals/configuration.rst

```diff
@@ -136,23 +135,23 @@ You configure options using the ``Configure<TOption>`` extension method. You can
   :dedent: 8
   :emphasize-lines: 7,10-13
 
-When you bind options to configuration each property in your options type is bound to a configuration key of the form ``property:subproperty:...``. For example, the ``MyOptions.Option1`` property is bound to the key ``Option1``, which is read from the ``option1`` property in *appsettings.json*. Note that configuration keys are case insensitive.
+When you bind options to configuration, each property in your options type is bound to a configuration key of the form ``property:subproperty:...``. For example, the ``MyOptions.Option1`` property is bound to the key ``Option1``, which is read from the ``option1`` property in *appsettings.json*. Note that configuration keys are case insensitive.
```

Comment:
Sorry, I was looking only at the diff as opposed to the whole document so I missed that title. Please allow me a few minutes to review all the text I haven't looked at.